tools/pipelines/test-azure-frs.yml

```diff
@@ -5,31 +5,24 @@
 
 name: $(Build.BuildId)
 
-trigger:
-  branches:
-    include:
-    - main
-    - next
-    - lts
-    - release/*
-  paths:
-    include:
-    - azure/packages/azure-client
-    - packages/framework/loader/container-loader
-    - packages/framework/fluid-static
-    - packages/drivers/routerlicious-driver
-    - packages/dds/map
-    - tools/pipelines/test-azure-frs.yml
-    - tools/pipelines/templates/include-test-real-service.yml
-
+trigger: none
 pr: none
 
 resources:
   pipelines:
   - pipeline: client
```

Comment:
I wonder if we should just remove this pipeline from this section... In the previous version of the code, it being here doesn't cause test-azure-frs to trigger. And after moving all paths for the CI trigger above to build-azure.yml, I don't see that this pipeline needs to reference Build - client packages at all.

Reply:
That would also solve the problem I mentioned in the description. If the only trigger for test-azure-frs is Build - azure, we can remove line 25 (`branch: main`) because the branch for which Build - azure triggered will "follow it" when test-azure-frs triggers in response.